site/docs/uk/guide/deployment-types.md

```diff
@@ -0,0 +1,417 @@
+|            |             *обидва чекаєте*               |           |
+|            |                                            |           |
+|  Telegram  |                                            |    Бот    |
+|            |                                            |           |
+|            |                                            |           |
+|            |    ---  привіт, нове повідомлення  --->    |           |
+|            |   <---    добре, sendChatAction     ---    |           |
+|____________|                                            |___________|
+```
+
+### Своєчасне завершення запитів вебхуків
+
+> Ви можете ігнорувати решту цієї сторінки, якщо всі ваші middleware завершуються швидко, тобто протягом кількох секунд.
+> Цей розділ насамперед призначений для людей, які хочуть виконувати передачу файлів у відповідь на повідомлення або інші операції, які потребують більше часу.
+
+Коли Telegram надсилає оновлення з одного чату вашому боту, він чекатиме, поки ви завершите запит, перш ніж доставити наступне оновлення, яке належить цьому чату.
+Іншими словами, Telegram послідовно надсилатиме оновлення з одного чату, а оновлення з різних чатів надсилатимуться одночасно.
+Джерело цієї інформації [тут](https://github.com/tdlib/telegram-bot-api/issues/75#issuecomment-755436496).
+
+Telegram намагається переконатися, що ваш бот отримує всі оновлення.
+Це означає, що якщо доставка оновлення для чату не вдасться, наступні оновлення будуть поставлені в чергу, доки перше оновлення не буде успішним.
+
+#### Чому не завершувати запит вебхуку небезпечно
+
+Telegram має тайм-аут для кожного оновлення, яке він надсилає на вашу кінцеву точку вебхуку.
+Якщо ви не завершите запит вебхуку досить швидко, Telegram повторно надішле оновлення, припускаючи, що воно не було доставлено.
+Унаслідок цього ваш бот може неочікувано обробити те саме оновлення кілька разів.
+Це означає, що він виконуватиме всю обробку оновлень, включаючи надсилання будь-яких відповідних повідомлень, кілька разів.
+
+```asciiart:no-line-numbers
+______________                                           _____________
+|            |                                           |           |
+|            | ---   привіт, нове повідомленнЯ   --->    |           |
+|            |                                      .    |           |
+|            |           *обробка ботом*            .    |           |
+|            |                                      .    |           |
+|  Telegram  | --- Я сказав нове повідомлення!!! --->    |    Бот    |
+|            |                                      ..   |           |
+|            |        *обробка ботом двічі*         ..   |           |
+|            |                                      ..   |           |
+|            | ---          ПРИИИВВВІІІТ         --->    |           |
+|            |                                      ...  |           |
+|            |       *обробка ботом тричі*          ...  |           |
```

Comment:
```suggestion
|            |        *обробка ботом двічі*         ..   |           |
|            |                                      ..   |           |
|            | ---          ПРИИИВВВІІІТ         --->    |           |
|            |                                      ...  |           |
|            |        *обробка ботом тричі*         ...  |           |
```

Centred the last text, and highlighted as much to see the last test from the middle.